Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -287,7 +287,7 @@ def read_info_file(filename):
         return None
 
 
-def write_info_file(filename, contents, overwrite=False):
+def write_into_file(filename, contents, overwrite=False):
```

Comment:
Maybe standardize the naming? You have a `read_file ` method, so:
```suggestion
def write_file(filename, contents, overwrite=False):
```